tests/engine/test_process_function.py

```diff
@@ -439,6 +439,10 @@ def test_launchers():
     assert result == get_true_node()
     assert isinstance(node, orm.CalcFunctionNode)
 
+
+@pytest.mark.requires_rmq
```

Comment:
This test fails in rabbitmq tests, I don't know why. Perhaps something else from `setup.sh` is needed? 

```
FAILED tests/engine/test_process_function.py::test_submit_launchers - kiwipy.exceptions.UnroutableError: ('NO_ROUTE', 'aiida-d201dba33ae749d8a63dce82958ca171.process.queue')
```
https://github.com/aiidateam/aiida-core/actions/runs/9289159100/job/25562353186?pr=6425

Reply:
It's now working, although I have no idea what I changed that made this work.

Reply:
This is just a transient problem that props up now and again on GHA.